pkg/daemon/client/client.go

```diff
@@ -25,7 +28,7 @@ type Client struct {
 
 // New initializes a new API client
 func New(endpoint string) *Client {
-	fmt.Printf("client initialized against addr: %s\n", endpoint)
+	logging.S().Infow("client initialized", "addr", endpoint)
```

Comment:
This is a bit ambiguous, because we have many clients:
1. testground client, talking to testground daemon
2. docker client, talking to docker daemon
3. kubectl client, talking to kubectl master

Additional context will be helpful, for example if we enable the caller (pkg, file, line number), or changing to log to: `testground client initialised`.